schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -60,4 +70,257 @@ public static void main(String... args) {
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
+      Properties configProperties,
+      @Nullable Path schemaFilePath,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Right<>(configProperties);
+    Either<Path, String> schema = new Left<>(schemaFilePath);
+    load(config, schema, options, createCoordinatorTable);
+  }
+
+  /**
+   * Creates tables defined in the schema file.
+   *
+   * @param configFilePath path to Scalar DB config file.
+   * @param schemaFilePath path to schema file.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
+      Path configFilePath,
+      @Nullable Path schemaFilePath,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Left<>(configFilePath);
+    Either<Path, String> schema = new Left<>(schemaFilePath);
+    load(config, schema, options, createCoordinatorTable);
+  }
+
+  /**
+   * Creates tables defined in the schema.
+   *
+   * @param configProperties Scalar DB config properties.
+   * @param serializedSchemaJson serialized json string schema.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
+      Properties configProperties,
+      @Nullable String serializedSchemaJson,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Right<>(configProperties);
+    Either<Path, String> schema = new Right<>(serializedSchemaJson);
+    load(config, schema, options, createCoordinatorTable);
+  }
+
+  /**
+   * Creates tables defined in the schema.
+   *
+   * @param configFilePath path to Scalar DB config file.
+   * @param serializedSchemaJson serialized json string schema.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
+      Path configFilePath,
+      @Nullable String serializedSchemaJson,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Left<>(configFilePath);
+    Either<Path, String> schema = new Right<>(serializedSchemaJson);
+    load(config, schema, options, createCoordinatorTable);
+  }
+
+  /**
+   * Creates tables defined in the schema file.
+   *
+   * @param config Scalar DB config.
+   * @param schema schema definition.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
```

Comment:
Let's make this method `private`:
```suggestion
  private static void load(
```